From 57f834becbbd6e61e517dbac06158228f46f5ceb Mon Sep 17 00:00:00 2001
From: Richard Henderson <richard.henderson@linaro.org>
Date: Fri, 1 May 2026 11:36:42 +1000
Subject: [PATCH] qemu: Enable FEAT_FPMR

---
 plat/qemu/common/common.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/plat/qemu/common/common.mk b/plat/qemu/common/common.mk
index aca125927..cde024cd9 100644
--- a/plat/qemu/common/common.mk
+++ b/plat/qemu/common/common.mk
@@ -131,6 +131,9 @@ ENABLE_FEAT_S1POE	:=	2
 # 9.3
 ENABLE_FEAT_GCS		:=	2
 
+# 9.5
+ENABLE_FEAT_FPMR	:=	2
+
 # SPM_MM is not compatible with ENABLE_SVE_FOR_NS (build breaks)
 ifeq (${SPM_MM},1)
 	ENABLE_SVE_FOR_NS	:= 0
-- 
2.43.0

